web/src/settings_org.js

```diff
@@ -258,9 +258,66 @@ function set_realm_waiting_period_setting() {
     );
 }
 
+function is_video_chat_provider_jitsi_meet() {
+    const video_chat_provider_id = Number.parseInt($("#id_realm_video_chat_provider").val(), 10);
+    const jitsi_meet_id = page_params.realm_available_video_chat_providers.jitsi_meet.id;
+    return video_chat_provider_id === jitsi_meet_id;
+}
+
+function get_jitsi_server_url_setting_value($input_elem, for_api_data = true) {
+    if (!is_video_chat_provider_jitsi_meet()) {
+        return page_params.realm_jitsi_server_url;
```

Comment:
This feels odd since we are passing the realm_jitsi_server_url even when video provider setting is not set to Jitsi. Would be better to add a comment explaining that we do this to make sure that the save button is hidden and API data does not contain the jitsi url parameter.